python/cuml/cluster/dbscan.pyx

```diff
@@ -168,18 +178,21 @@ class DBSCAN:
 
         input_ptr = self._get_ctype_ptr(X_m)
 
+        cdef cumlHandle* handle_ = <cumlHandle*><size_t>self.handle.getHandle()
         self.labels_ = cudf.Series(np.zeros(self.n_rows, dtype=np.int32))
         self.labels_array = self.labels_._column._data.to_gpu_array()
         cdef uintptr_t labels_ptr = self._get_ctype_ptr(self.labels_array)
         if self.gdf_datatype.type == np.float32:
-            dbscanFit(<float*>input_ptr,
+            dbscanFit(handle_[0],
+                      <float*>input_ptr,
                                <int> self.n_rows,
                                <int> self.n_cols,
                                <float> self.eps,
                                <int> self.min_samples,
                        <int*> labels_ptr)
         else:
-            dbscanFit(<double*>input_ptr,
+            dbscanFit(handle_[0],
+                      <double*>input_ptr,
```

Comment:
I can't leave a comment on the unchanged line 201 so putting it here. In line 201 `del(X_m)` is called. Since `dbscanFit` does not guarantee to have all work done before it finishes is there a sync needed before `del(X_m)` or does that call ensure the necessary synchronization?

Reply:
good catch! Added the sync call here.